src/layers/TrajservLayer.js

```diff
@@ -34,6 +36,53 @@ class TrajservLayer extends TrackerLayer {
     return newDates;
   }
 
+  /**
+   *  Translate the response date object into a readable object.
+   * @param {ol.feature} feature openlayer ol.feature.
+   * @param {string} color color hex string.
+   * @returns {array<Date>}
+   */
+  static getTrajectoryStyle(feature, color) {
```

Comment:
give the style to the feature directly so you donm' have to create this function.